mcnp2cad.cpp

```diff
@@ -153,7 +153,14 @@ class GeometryContext {
   std::string materialName( int mat, double rho ){
     std::string ret;
     std::stringstream formatter;
-    formatter << "mat_" << mat << "_rho_" << rho;
+    if(Gopt.uwuw_names){
+      char buffer [50];
+      int n;
+      n = sprintf(buffer, "%E", rho);
```

Comment:
My reason for the ```sprintf``` was to make the string formatting more reproducible across multiple languages, i.e. Python. This isn't a huge deal, but I could imagine it saving headaches in some situation in the future.